### ISSUE TITLE
Set own tags

### ISSUE THREAD

**Issue by HyperCriSiS** [a-tak/auto-bucket#157]
Hi,

why is it not possible to set own tags?

Best regards,
Hyper

**Comment by sverley** [comment]
Yes, you can. Go to Option/Classification tag setting and select tags you want

**Comment by JosephCatrambone** [comment]
Apologies for jumping in so late, but I just found this plugin and am wondering the same thing.  If it's not clear, one needs to add the tags to their mail messages first, then restart thunderbird:

Right click on a message, go to 'tags', and then 'add new tag'.

https://user-images.githubusercontent.com/2160055/206387603-cbed77c6-0bdf-4f3e-9acc-33505be86342.mp4

After you've added your custom tags, restart thunderbird and then the tags will appear under Options > Classification Tag.

**Comment by HyperCriSiS** [comment]
Thanks!
I like the general idea of this, but Popfile still seems still the most convenient for me.
Unfortunately there is no Thunderbird addon available and the the development stopped ages ago :slightly_frowning_face:
This is why I was searching for an alternative, but the thing with the tags is not handy.

What do you think about making it folder based? So you just click on a folder and assign it to auto-bucket which learns from all E-Mails in it. New incoming E-Mails which fit the folder get sorted.
If a Mail is moved to another folder it automatically relearns.
That would be so awesome!

What do you think? @sverley
